wxcrafter/src/wxgui_bitmaploader.cpp

```diff
@@ -14,7 +14,12 @@ wxCrafter::ResourceLoader::ResourceLoader(const wxString& skin)
 {
     if(m_bitmaps.empty()) {
         wxString zipFile;
+#ifdef USE_POSIX_LAYOUT
+        zipFile << wxStandardPaths::Get().GetDataDir() + wxT(INSTALL_DIR) << wxFileName::GetPathSeparator() << skin
```

Comment:
Strange to mix `+` and `<<`

Reply:
I can change the + to << if you think that is more correct or common.
I will test it and verify it works.